### PR TITLE
Cleanup electricity-trade example docker containers and fix readme. 

### PR DESCRIPTION
**Pull request by rybalko**

### CODE REVIEW

**Comment by petermetz** [comment]
@rybalko Thank you very much for the contribution! Please do the following:
1. squash to a single commit
2. write the commit message according to the conventional commits guide lines (see CONTRIBUTING.md)
3. rebase onto the latest main
4. fix the DCO (see CONTRIBUTING.md)

File: examples/electricity-trade/cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+docker stop sawtooth-shell-default
+docker stop sawtooth-settings-tp-default
+docker stop sawtooth-xo-tp-python-default
+docker stop sawtooth-devmode-engine-rust-default
+docker stop sawtooth-rest-api-default
+docker stop sawtooth-intkey-tp-python-default
+docker stop sawtooth-validator-default
+docker stop geth1
+docker stop gethtestnet_init-chain-geth1_run_1
```
Comment (takeutak) [inline examples/electricity-trade/cleanup.sh:9]: Would you remove "docker stop gethtestnet_init-chain-geth1_run_1"? This container will be stopped when geth1 container is launched.

File: examples/electricity-trade/cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+docker stop gethtestnet_init-chain-geth1_run_2
```
Comment (takeutak) [inline examples/electricity-trade/cleanup.sh:10]: Would you remove "docker stop gethtestnet_init-chain-geth1_run_2"? The reason is like the above.

File: examples/electricity-trade/cleanup.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+docker rm sawtooth-shell-default
+docker rm sawtooth-settings-tp-default
+docker rm sawtooth-xo-tp-python-default
+docker rm sawtooth-devmode-engine-rust-default
+docker rm sawtooth-rest-api-default
+docker rm sawtooth-intkey-tp-python-default
+docker rm sawtooth-validator-default
+docker rm geth1
+docker rm gethtestnet_init-chain-geth1_run_1
```
Comment (takeutak) [inline examples/electricity-trade/cleanup.sh:20]: Would you remove "docker rm gethtestnet_init-chain-geth1_run_1"? The reason is like the above.

File: examples/electricity-trade/cleanup.sh [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+docker rm gethtestnet_init-chain-geth1_run_2
+
+sudo rm -R ../../tools/docker/geth-testnet/data-geth1/geth
+sudo rm ../../tools/docker/geth-testnet/data-geth1/geth.ipc*
```
Comment (takeutak) [inline examples/electricity-trade/cleanup.sh:24]: This is only a comment (not a change request) that I'm wondering that the error message "rm: cannot remove '../../tools/docker/geth-testnet/data-geth1/geth.ipc*': No such file or directory" happens (but the file ../../tools/docker/geth-testnet/data-geth1/geth.ipc is removed by this script)
I will accept this change once it is harmless.  If an improvement is found, this part may be fixed on later pull-requests.

**Comment by takeutak** [comment]
@rybalko Thank you for your intelligent commit to our code! I think your PR is helpful for us.
For the code itself, I appreciate if you could consider about my above points.
Also, as @petermetz said, there are some manner of pull-request in Cactus. So could you reflect the @petermetz 's points?
It may be difficult, but if you are in trouble, please leave a comment. I will help you then. Good luck!

**Comment by rybalko** [comment]
Hello @takeutak 
perhaps it is better to cancel this pull request then and I'll revert my commits to our opentaps/cactus repo
and made new one with readme changes and cleanup.sh and correct description

about
sudo rm -R ../../tools/docker/geth-testnet/data-geth1/geth
sudo rm ../../tools/docker/geth-testnet/data-geth1/geth.ipc*
I just noticed during start/stop electricity-trade example several times 
that it could keeps those directory and file, and they are with 'root' access
what is cause error in next run
so I decided clean up them

**Comment by takeutak** [comment]
@rybalko Thanks! 

> perhaps it is better to cancel this pull request then and I'll revert my commits to our opentaps/cactus repo
and made new one with readme changes and cleanup.sh and correct description

That's sounds great! I'm looking forward to your new pull-request.

> about
> sudo rm -R ../../tools/docker/geth-testnet/data-geth1/geth
> sudo rm ../../tools/docker/geth-testnet/data-geth1/geth.ipc*
> I just noticed during start/stop electricity-trade example several times
> that it could keeps those directory and file, and they are with 'root' access
> what is cause error in next run
> so I decided clean up them

Great! As you said, I agree with your idea that cleanup.sh includes the above commands.

I think it is a good Pull-request, so I would like to ask you to post it. If you have any trouble, please post a comment. Good luck!

**Review by petermetz — CHANGES_REQUESTED** [review]
@rybalko See my comment above for the changes requested from my side.

**Comment by petermetz** [comment]
@rybalko For future reference: You don't need to close/recreate PRs from scratch, you can also just force push to your branch instead when history needs to be altered.

**Pull request closed without merging**